src/js/utils/DefaultErrorUtil.tsx

```diff
@@ -0,0 +1,38 @@
+import { i18nMark } from "@lingui/react";
+
+// @ts-ignore
+import {
+  ToastNotification,
+  ToastAppearance
+} from "@extension-kid/toast-notifications";
+
+import container from "#SRC/js/container";
+import {
+  NotificationService,
+  NotificationServiceType
+} from "@extension-kid/notification-service";
+
+const notificationService = container.get(
+  NotificationServiceType
+) as NotificationService;
+
+const generateDefaultNetworkErrorHandler = (i18n: any) => {
```

Comment:
since you already have `container` imported you could get the i18n object from it.

```javascript
import { I18n } from "@lingui/core";
import { TYPES } from "#SRC/js/types/containerTypes";
const i18n = container.get<I18n>(TYPES.I18n);
```